src/com/facebook/buck/android/FilterResourcesSteps.java

```diff
@@ -68,7 +68,7 @@
 
   @VisibleForTesting
   static final Pattern NON_ENGLISH_STRINGS_FILE_PATH =
-      Pattern.compile("\\b|.*/res/values-([a-z]{2})(?:-r([A-Z]{2}))*/strings.xml");
+      Pattern.compile("\\b|.*/res/values-([a-z]{2})(?:-r([A-Z]{2}))*/.*.xml");
```

Comment:
can it really be absolutely anything and not just something like `[w+-]`?

Reply:
found an answer in https://developer.android.com/guide/topics/resources/providing-resources.html:
```
Because each resource is defined with its own XML element, you can name the file whatever you want and place different resource types in one file.
```

It's weird but oh well...